### PR TITLE
chore: update media-player, ui-connected, and bugsnag config

### PR DESCRIPTION
**Pull request by nlewis84**
## New Info (current)
This updates ui-connected and media player, while also fixing an issue with bugsnag that was causing Android crashes. This started as a try at fixing their broken Resi videos on Android. These package updates did NOT solve this problem, but it did reveal an issue with bugsnag that is addressed with this PR.

## Old Info (out of date)
This bumps packages so that Resi videos will correctly play on Android devices. Currently not able to build a working version of the app on simulator or on a device due to a crashing bug with BugSnag.

API key exists in the APP .env. 

Screen Shot 2022-02-07 at 11 52 18 AM

### CODE REVIEW

**Comment by nlewis84** [comment]
@redreceipt @conrad-vanl Updating these packages did not fix the HLS issue on Android.

**Comment by nlewis84** [comment]
I have working Android builds now (fixed the bugsnag issue). Probably some code got deleted by accident on an app upgrade.

File: apolloschurchapp/Gemfile.lock [hunk 1]
```diff
@@ -275,6 +275,7 @@ GEM
 
 PLATFORMS
   x86_64-darwin-20
+  x86_64-darwin-21
```
Comment (nlewis84) [inline apolloschurchapp/Gemfile.lock:278]: @redreceipt This is the only change that I'm questioning. I was getting failed builds on sim when I tried to remove this. Yarn was adding this line automatically.

Comment (redreceipt) [reply apolloschurchapp/Gemfile.lock:278]: yea just leave it. won't hurt. not sure why it's added sometimes.

**Review by redreceipt — APPROVED** [review]

**Comment by conrad-vanl** [comment]
should look like this on android

**Comment by nlewis84** [comment]
Okay, there must be another issue then. This is what it looks like with these packages upgraded.
Screenshot_1645481960

**Comment by conrad-vanl** [comment]
You might debug these lines then local to chase oaks:
https://github.com/ApollosProject/apollos-apps/blob/master/packages/apollos-ui-connected/src/NodeSingleConnected/index.js#L110

https://github.com/ApollosProject/apollos-apps/blob/master/packages/apollos-ui-media-player/src/useSourceComponents.tsx#L43

**Comment by nlewis84** [comment]
https://github.com/ApollosProject/apollos-apps/blob/master/packages/apollos-ui-connected/src/NodeSingleConnected/index.js#L110

This code isn't getting touched, so `video.embedHtml` is never transferred to `sources.html` and `sources.html` is undefined. Been trying to track down the breakdown, but no luck yet.

**Comment by conrad-vanl** [comment]
I bet you just need to update `@apollosproject/ui-fragments` as well - this attribute was added to that query - https://github.com/ApollosProject/apollos-apps/commit/313008eddbfe52e0abad1eee4e073d2a780c25ab#diff-b50254d77da603ede8d6049b8d125a114e7c4b5d9755799a8787b22feb511c19R20

**Comment by nlewis84** [comment]
Dang, you are right...I thought I identified all the changed packages, but missed that one. Unfortunately, I only have working audio on android devices with these package updates.

**Comment by conrad-vanl** [comment]
Hmmmm... can you see if it's any different when running Storybook on core? On core the videos were playing fine for me in storybook on android emulator. Wondering if it's an emulator problem on your machine, and if so, we should still release this. If video IS working for you on core, then there might be something goofy with the native files on chase oaks we need to fix.

**Comment by nlewis84** [comment]
Okay, I confirmed this is working on actual devices, but not on simulators (only audio and a still image happens on sim). I'm going to go ahead and promote this new version to the app store to get the change out into the wild.

**Pull request merged**